hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VElement<{}>[]
+): VNode<Data>
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  ...children: VElement<{}>[]
+): VNode<Data>
+
```

Comment:
@Swizz Can we just leave out the `...children`? h() supports ...children just because of JSX. If JSX didn't exist, h() would be always Array of VNode.